script/deploy-launch.sol

```diff
@@ -8,22 +8,22 @@ import "openzeppelin/utils/Strings.sol";
 
 import "openzeppelin/proxy/ERC1967/ERC1967Proxy.sol";
 
-import "../src/core/OptionToken.sol";
```

Comment:
@antoncoding can you keep the relative paths. When projects brings this repo in as a submodule, vscode doesnt recognize the paths. Compilation still works but its not a great experience in the ide with all the path resolution issues.

Reply:
In fact i am getting compilation errors when trying to run cross-margin-engine tests.

Reply:
it tries to look for things files from the root of the parent contract and if those files do exist the content is different. It all falls apart.